core/src/main/java/co/aikar/commands/RegisteredCommand.java

```diff
@@ -341,7 +333,21 @@ public String getPrefSubCommand() {
     }
 
     public String getSyntaxText() {
-        return syntaxText;
+        return getSyntaxText(null);
+    }
+
+    public String getSyntaxText(CommandIssuer issuer) {
+        if (syntaxText != null) return syntaxText;
+        StringBuilder syntaxBuilder = new StringBuilder(64);
+        for (CommandParameter<?> parameter : parameters) {
+            if (parameter.getSyntax(null) != null) {
```

Comment:
Why call it for null and then recall it with issuer? can't this work with just 1 call?